Assets/_Functionalities/ObjImporter/Scripts/OBJSpawner.cs

```diff
@@ -109,9 +109,6 @@ private void OnObjImported(GameObject returnedGameObject)
             importedObject = returnedGameObject;
             returnedGameObject.AddComponent<MeshCollider>();
             DisposeImporter();
-            
-            // Object is loaded / replaced - trigger the application of styling
-            holgo.ApplyStyling();
```

Comment:
Dit doet niet de styling ongedaan maken - de kleur is nog steeds aanwezig in de LayerData.Styling; hier wordt alleen het toepassen van alle styling na import verwijderd. Dit kan bij verder gebruik onduidelijke bugs opleveren